assets/wire/components/WireLabel.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from  'classnames';
+import {IArticle} from 'interfaces';
+import {wireLabel} from 'agenda/utils';
+
+interface IProps {
+    item: IArticle;
+}
+
+export default function WireLabel({item}: IProps) {
+    const classes = classNames('label label--fill label--rounded');
+
+    const subjectList = item.subject || [];
+    const wireItemSubjects = subjectList.filter(wireLabel);
+
+    return (
+        wireItemSubjects.length > 0 ? (
+            <div>
+                {wireItemSubjects.map(subject => (
+                    <span
+                        className={classes + ` label-wire--${subject.code}`}
+                        key={subject.name}
+                    >
+                        {subject.name}
```

Comment:
we will need to translate those unlike the other subjects, so pls use `{gettext(subject.name)}` here (the translations will be populated based on backend code)

Reply:
how would `gettext` help in translating strings that sit in the DB?

Reply:
those strings will be populated on incoming items via some backend code, so we will have them available for extraction and then later for usage

Reply:
> we will have them available for extraction and then later for usage

would marking for extraction happen on the back-end? Maybe we could return the translated ones from back-end too?

Reply:
yep it would, but to return it translated based on user language would require some processing when it's fetched and that would be quite complicated